.circleci/continue_config.yml

```diff
@@ -2757,6 +2757,19 @@ jobs:
             cd appsec/build
             make -j $(nproc) ddappsec_helper_test
             ./tests/helper/ddappsec_helper_test
+      - when:
+          # Copy ruleset only on x86_64
```

Comment:
Why that? :-)

Reply:
I guess I was trying to avoid downloading it twice, but I changed it now since it makes the packaging script simpler.